messagebus/messagebus.go

```diff
@@ -40,8 +40,11 @@ type Message struct {
 	PrivateInstanceId   string            `json:"private_instance_id"`
 	ServerCertDomainSAN string            `json:"server_cert_domain_san,omitempty"`
 	AvailabilityZone    string            `json:"availability_zone,omitempty"`
+	Options             map[string]string `json:"options,omitempty"`
 }
 
+const LoadBalancingAlgorithm string = "lb_algo"
```

Comment:
here you call it `lb_algo` but in README.md you called it `lb_algorithm` - I'm very much in favor of `lb_algo` (or `lb_alg`) because it is shorter on the NATS message. but please stick to one version in both cases

Reply:
As I wrote above, I would keep "lb_algo"